frontend/src/components/layout/navigation/SignInButton.module.scss

```diff
@@ -0,0 +1,14 @@
+@import "../../../styles/tokens.scss";
+
+.sign-in-button {
+  // We're using CSS's `currentColor` here because the colour
+  // is different in the light (Premium) and dark (regular) themes:
+  border: 2px solid currentColor;
+  border-radius: $border-radius-sm;
+  padding: $spacing-xs $spacing-md;
+  height: unset;
```

Comment:
Generally, I'd prefer not to add styles in one place only to remove them in another, as that makes it hard for future contributors to predict what effect changing the styles in the former will have elsewhere.

Instead, I'd prefer being more specific with the rule that applies the rule in the first place. In this case, I think that you'll probably want the `height: 100%` that's set on `.link` in `Navigation.module.scss` to instead only be set on `.link.is-active` elements?

This has the added benefit of avoiding an interesting bug (or just weirdness?) in CSS modules that you're running into: the order of the styles in the dev build (`npm run dev`) appear to be different from the one in the production build (`npm run watch`), which means that the styles also get applied in a different order, and hence the `unset`ting not working in the production build:

![image](https://user-images.githubusercontent.com/4251/170032051-f3c79c8b-2414-443d-b28b-005246bb29c4.png)

![image](https://user-images.githubusercontent.com/4251/170032167-b5145880-3040-47d6-936e-fbcf04dd58ba.png)